src/V3EmitCFunc.cpp

```diff
@@ -633,6 +633,17 @@ void EmitCFunc::emitVarReset(AstVar* varp) {
                 emitSetVarConstant(varNameProtected + ".at(" + cvtToStr(itr.first) + ")",
                                    VN_AS(valuep, Const));
             }
+        } else if (AstWildcardArrayDType* const adtypep = VN_CAST(dtypep, WildcardArrayDType)) {
+            if (initarp->defaultp()) {
+                emitSetVarConstant(varNameProtected + ".atDefault()",
```

Comment:
You need to add setting and using the "default" in the wildcard test, so this code is covered.

Reply:
This code path is tested by assigning `'{..., default: ...}` in the t_assoc_wildcard test.